<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileActionDialog</class>
 <widget class="QDialog" name="FileActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>242</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>242</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>242</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Target file already exists!</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTarget">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Overwrite:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTargetFile">
       <property name="text">
        <string notr="true">c:\huhu\test.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTargetFileInfos">
       <property name="text">
        <string>12345 bytes, 01.01.1970 00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSource">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>With:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSourceFile">
       <property name="text">
        <string notr="true">c:\huhu\test.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSourceFileInfos">
       <property name="text">
        <string>54321 bytes, 12.12.1980 11:11:11</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonOverwrite">
         <property name="text">
          <string>Overwrite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSkip">
         <property name="text">
          <string>Skip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonOverwriteAll">
         <property name="text">
          <string>Overwrite all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSkipAll">
         <property name="text">
          <string>Skip all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxOptions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>122</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>122</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
